examples/gradle-kt-dsl/lib/build.gradle.kts

```diff
@@ -0,0 +1,33 @@
+val accpVersion = "1.6.1"
+val archEnv = System.getenv("ARCH")?: "x86_64"
```

Comment:
can we default to system-defined architecture instead?

https://stackoverflow.com/a/29508308

Reply:
The issue with "os.arch" is that it's not very clear what the possible values are. So instead I opted for the option of explicitly passing it to the script via an environment variable.

Reply:
ok, sounds good